exercises/practice/allergies/.meta/example.f90

```diff
@@ -0,0 +1,76 @@
+
+module allergies
+  implicit none
+
+  integer,parameter :: ALLERGY_TYPES_LEN=8
+
+  integer, parameter :: unknown =0
+  integer, parameter :: eggs =1
+  integer, parameter :: peanuts =2
+  integer, parameter :: shellfish =3
+  integer, parameter :: strawberries =4
+  integer, parameter :: tomatoes =5
+  integer, parameter :: chocolate =6
+  integer, parameter :: pollen =7
+  integer, parameter :: cats =8
```

Comment:
Style change from `somehing=value` to `something =value` when later we see syntax like `something = value`.  What is the significance of the style change?  What is being communicated for the difference in communication?  I don't see the subtle communication making sense here.

Reply:
Agreed, did not review my own commit